### DISCUSSION TITLE
Request timeout in sendVideo

### DISCUSSION THREAD

**Discussion by DimaDDM** [eternnoir/pyTelegramBotAPI#2054] [Q&A]
Sometimes timeout error appears when i'm trying send medium size video files (~25mb)
`Traceback (most recent call last):
    await self.bot.send_video(chat_id, media_item, caption=caption)
  File "/usr/local/lib/python3.10/site-packages/telebot/async_telebot.py", line 3153, in send_video
    await asyncio_helper.send_video(
  File "/usr/local/lib/python3.10/site-packages/telebot/asyncio_helper.py", line 713, in send_video
    return await _process_request(token, method_url, params=payload, files=files, method='post')
  File "/usr/local/lib/python3.10/site-packages/telebot/asyncio_helper.py", line 105, in _process_request
    raise RequestTimeout("Request timeout. Request: method={0} url={1} params={2} files={3} request_timeout={4}".format(method, url, params, files, request_timeout, current_try))
telebot.asyncio_helper.RequestTimeout: Request timeout. Request: method=post url=sendVideo params=<aiohttp.formdata.FormData object at 0x7fb17c0d25f0> files={'video': <telebot.types.InputMediaVideo object at 0x7fb17c0d1e10>} request_timeout=300
`
download video method

```
dbytes, dsize = await self.__download_file(media_url, session)
InputMediaVideo(dbytes)

    async def __download_file(self, url, session):
        web_data = await session.get(url)
        data_bytes = await web_data.content.read()
        size = len(data_bytes)
        return data_bytes, size

```

1. What version of pyTelegramBotAPI are you using?
4.13

2. What OS are you using?
Linux, Ubuntu

3. What version of python are you using?
3.10

**Comment by Badiboy** [comment]
So?

**Comment by DimaDDM** [comment]
> So?

The sendVideo method does not execute properly, although if you send without the library, the download and upload work. Is it possible to increase the timeout, add timeout adjustments, or fix a bug for medium-sized videos?

**Comment by Badiboy** [comment]
You can set any timeout you want... Globally or for every command...

**Reply by DimaDDM** [reply]
This error appears before time is out!

**Comment by Badiboy** [comment]
_before time is out!_

Any proof? Do not see any timeout set in code. Do not see any time markers.
